ksql-functional-tests/src/test/resources/query-validation-tests/table-functions.json

```diff
@@ -0,0 +1,102 @@
+{
+  "comments": [
+    "Tests applicable to table functions in general - we use EXPLODE here but we could use any table table function"
+  ],
+  "tests": [
```

Comment:
let's add a test for this with another UDF call (both unaliased)

Reply:
Added